app/controllers/api/account.php

```diff
@@ -3703,6 +3706,99 @@
         $session->setAttribute('factors', $factors);
         $dbForProject->updateDocument('sessions', $session->getId(), $session);
 
+        // Send email to user informing that MFA is enabled
+        $subject = $locale->getText("emails.mfaCreated.subject");
+        $customTemplate = $project->getAttribute('templates', [])['email.mfaCreated-' . $locale->default] ?? [];
+
+        // Appwrite theming for console
+        if ($project->getInternalId() === 'console') {
+            $message = Template::fromFile(__DIR__ . '/../../config/locale/templates/email-base-styled.tpl');
+            $body = Template::fromFile(__DIR__ . '/../../config/locale/templates/email-mfa-altered.tpl');
+
+            $body
+                ->setParam('{{buttonText}}', $locale->getText("emails.mfaCreated.buttonText"))
+                ->setParam('{{redirect}}', 'http://' . App::getEnv('_APP_DOMAIN') . '/console/account')
```

Comment:
Might be good to be consistent and use the same approach we do in other places like:

https://github.com/appwrite/appwrite/blob/5ddc705099a7751b12432f37d1b0f0381c9aecfa/app/controllers/api/account.php#L1072-L1073

Reply:
Updated 👍